pkg/plugins/helm/v1/api.go

```diff
@@ -193,6 +193,7 @@ func (p *createAPISubcommand) InjectResource(res *resource.Resource) error {
 	}
 
 	// Selected CRD version must match existing CRD versions.
+	// nolint:staticcheck
```

Comment:
This is a temporary fix, should be removed after [this](https://github.com/kubernetes-sigs/kubebuilder/pull/3261) is merged.

Reply:
@varshaprasad96 Sounds good, I'll create an issue and link it here.

Reply:
Based on my offline discussion with @varshaprasad96, we don't need an issue for this and we agreed to proceed with the `no:linter` check for now.